tests/e2e/specs/backend/single-product-details.test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import {
+	canvas,
+	createNewPost,
+	switchUserToAdmin,
+} from '@wordpress/e2e-test-utils';
+import { searchForBlock } from '@wordpress/e2e-test-utils/build/inserter';
+
+/**
+ * Internal dependencies
+ */
+import {
+	filterCurrentBlocks,
+	goToTemplateEditor,
+	insertBlockDontWaitForInsertClose,
+	useTheme,
+} from '../../utils.js';
+
+const block = {
+	name: 'Product Details',
+	slug: 'woocommerce/product-details',
+	class: '.wp-block-woocommerce-product-details',
+};
+
+describe( `${ block.name } Block`, () => {
+	it( 'in can not be inserted in a post', async () => {
+		await switchUserToAdmin();
+		await createNewPost( {
+			postType: 'post',
+			title: block.name,
+		} );
+		await searchForBlock( block.name );
+		expect( page ).toMatch( 'No results found.' );
+	} );
+
+	describe( 'in FSE editor', () => {
+		useTheme( 'emptytheme' );
+
+		beforeEach( async () => {
+			await goToTemplateEditor( {
+				postId: 'woocommerce/woocommerce//single-product',
+			} );
+		} );
+
+		it( 'can be inserted in FSE area', async () => {
+			await insertBlockDontWaitForInsertClose( block.name );
+			await expect( canvas() ).toMatchElement( block.class );
+		} );
+
+		it( 'can be inserted more than once', async () => {
+			await insertBlockDontWaitForInsertClose( block.name );
+			await insertBlockDontWaitForInsertClose( block.name );
+			const foo = await filterCurrentBlocks(
```

Comment:
Could we change the name of the variable? I guess this was copy-pasted from another test, I think I was actually the one that _temporarily_ created it and forgot to change the name 🤦‍♀️

Reply:
Good catch, I completely missed it! I fixed it in this test and all other tests which were using this variable name (https://github.com/woocommerce/woocommerce-blocks/pull/8470/commits/4f029a8660b465ca8e091635434e69a382940a0a). I also took the change to fix a typo in all those tests as well (https://github.com/woocommerce/woocommerce-blocks/pull/8470/commits/8d4e22cdc65e09f9533d9bc19f90201bdcbb38e2).

Reply:
Thanks! 🙌